x/masterchef/client/cli/tx_cmds.go

```diff
@@ -280,7 +280,7 @@ $ %s tx masterchef claim-rewards --from mykey --pool-ids [pool-ids]
 			poolIds := []uint64{}
 
 			poolIdsString, err := cmd.Flags().GetString(FlagPoolIds)
-			if err != nil {
+			if err == nil && poolIdsString != "" {
```

Comment:
could you please add tests too ? Looks like this command was never used

Reply:
There is no unit test for the whole package, and adding that will have to deprioritize leveragelp. This is cli cmd only, so will add it later on.

Reply:
okay, please add a task for it